spec/exponential_backoff_spec.rb

```diff
@@ -26,53 +26,29 @@
   end
 
   it "generates 10 randomized intervals" do
-    expect(subject.new(tries: 9).intervals).must_equal([
-      0.5244067512211441,
-      0.9113920238761231,
-      1.2406087918999114,
-      1.7632403621664823,
-      2.338001204738311,
-      4.350816718580626,
-      5.339852157217869,
-      11.889873261212443,
-      18.756037881636484,
-    ])
+    expect(subject.new(tries: 9).intervals.map { |x| x.round(3) }).must_equal(
+      [0.524, 0.911, 1.241, 1.763, 2.338, 4.351, 5.34, 11.89, 18.756]
```

Comment:
Apparently rubinius has totally different floating point noise... so the build fails on the way out digits